build.gradle

```diff
@@ -106,133 +165,21 @@ task scriptingJavadocsJson(type: Exec) {
 
 tasks.processResources.dependsOn(scriptingJavadocsJson)
 
-/* Comparing keys */
-task compareLangFiles {
```

Comment:
This is not acceptable. How did you manage to build the lang files without these tasks?

Reply:
I see now that mappet, unlike most of the other McHorse mods, does not have yaml files, I misunderstood the purpose of these tasks... Looks like they are not needed for building lang files. Nonetheless, these tasks should not be removed. There was no need for removal. Please don't just remove things from the existing working system.